securedrop_client/models.py

```diff
@@ -17,6 +19,79 @@
 Base = declarative_base()
 
 
+class Source(Base):
+    __tablename__ = 'sources'
```

Comment:
can you explain why you recommend we use autoincrement? from [the SQLite docs](https://www.sqlite.org/autoinc.html):

> The AUTOINCREMENT keyword imposes extra CPU, memory, disk space, and disk I/O overhead and should be avoided if not strictly needed. It is usually not needed.

Reply:
I was assuming we do that to move that to the database level instead of the ORM level. Like, if this was postgres, we'd force the autoincrement into the DB too. Idk. Maybe this isn't needed.